2017/03/20 18:26:54 [emerg] 1#1: "http" directive is not allowed here in /etc/nginx/conf.d/Jiller.conf:1
2017/03/20 18:36:20 [error] 7#7: *1 open() "/media/notes/404_bZT1fbY.jpg" failed (2: No such file or directory), client: 172.19.0.1, server: localhost, request: "GET /media/notes/404_bZT1fbY.jpg HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/note/"
2017/03/20 18:36:20 [error] 7#7: *1 open() "/media/notes/voolf_UyWYHOe.jpg" failed (2: No such file or directory), client: 172.19.0.1, server: localhost, request: "GET /media/notes/voolf_UyWYHOe.jpg HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/note/"
2017/03/24 12:48:38 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.19.0.6:8000/", host: "127.0.0.1"
2017/03/24 12:48:44 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.19.0.6:8000/favicon.ico", host: "127.0.0.1"
2017/03/24 12:48:48 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.19.0.6:8000/", host: "127.0.0.1"
2017/03/24 12:50:01 [error] 7#7: *7 open() "/media/notes/bug_time.png" failed (2: No such file or directory), client: 172.19.0.1, server: localhost, request: "GET /media/notes/bug_time.png HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/note/"
2017/03/24 12:50:01 [error] 7#7: *8 open() "/media/notes/bugs_and_features.jpg" failed (2: No such file or directory), client: 172.19.0.1, server: localhost, request: "GET /media/notes/bugs_and_features.jpg HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/note/"
2017/03/24 12:50:01 [error] 7#7: *1 open() "/media/notes/IMG_20170307_154004.jpg" failed (2: No such file or directory), client: 172.19.0.1, server: localhost, request: "GET /media/notes/IMG_20170307_154004.jpg HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/note/"
2017/03/24 12:52:53 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /project/1/note/ HTTP/1.1", upstream: "http://172.19.0.6:8000/project/1/note/", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/sprint/"
2017/03/24 12:52:57 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /project/1/note/ HTTP/1.1", upstream: "http://172.19.0.6:8000/project/1/note/", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/sprint/"
2017/03/24 12:52:58 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /project/1/note/ HTTP/1.1", upstream: "http://172.19.0.6:8000/project/1/note/", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/sprint/"
2017/03/24 12:52:59 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /project/1/note/ HTTP/1.1", upstream: "http://172.19.0.6:8000/project/1/note/", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/sprint/"
2017/03/24 12:53:01 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /project/1/note/ HTTP/1.1", upstream: "http://172.19.0.6:8000/project/1/note/", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/sprint/"
2017/03/24 12:53:02 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /project/1/note/ HTTP/1.1", upstream: "http://172.19.0.6:8000/project/1/note/", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/sprint/"
2017/03/24 12:53:05 [error] 7#7: *11 open() "/media/notes/bugs_and_features.jpg" failed (2: No such file or directory), client: 172.19.0.1, server: localhost, request: "GET /media/notes/bugs_and_features.jpg HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/note/"
2017/03/24 12:53:05 [error] 7#7: *1 open() "/media/notes/bug_time.png" failed (2: No such file or directory), client: 172.19.0.1, server: localhost, request: "GET /media/notes/bug_time.png HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/note/"
2017/03/24 12:53:05 [error] 7#7: *9 open() "/media/notes/IMG_20170307_154004.jpg" failed (2: No such file or directory), client: 172.19.0.1, server: localhost, request: "GET /media/notes/IMG_20170307_154004.jpg HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/project/1/note/"
2017/03/24 13:47:13 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.19.0.6:8000/", host: "127.0.0.1"
2017/03/24 13:47:13 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.19.0.6:8000/", host: "127.0.0.1"
